uber/templates/jobs/form.html

```diff
@@ -53,41 +53,43 @@
     <td> <b>Start Time:</b> </td>
     <td>
         <select name="start_time">
-            {% if job.type == c.SETUP %}
-		<!-- SETUP -->
-                {% options c.SETUP_TIME_OPTS job.start_time %}
-            {% elif job.type == c.TEARDOWN %}
-		<!-- TEARDOWN -->
-		{% options c.TEARDOWN_TIME_OPTS job.start_time %}
-            {% elif job.type == c.REGULAR %}
-		<!-- REGULAR -->
-		{% options c.START_TIME_OPTS job.start_time %}
+
+        {% if job.type == c.SETUP %}
```

Comment:
This looks like some whitespace got munged. Let's set this back up to be consistent for readability.